_posts/2025-06-04-cockpit-340.md

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
This seems to be the wrong commit? We discussed 340 in https://github.com/cockpit-project/cockpit-project.github.io/pull/802 and merged https://github.com/cockpit-project/cockpit-project.github.io/commit/dba26a6f343b06785c386e69fdf586f05f1eb0af

I guess you still had this in your own main and that's why its in the pull request?

Reply:
Woops, yeah you're right. This should only be about the workflow